code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -583,12 +583,67 @@
 	item_state = "gold_id"
 	registered_name = "Captain"
 	assignment = "Captain"
+	var/list/locked_access
+	var/fLocked = TRUE
 
+/obj/item/weapon/card/id/captains_spare/summoned
```

Comment:
you don't actually have to put a separate definition here if you're not changing any vars, although you might add a cheeky description like `This one looks extra magical.`